tests/53-sim-binary_tree.c

```diff
@@ -29,9 +29,53 @@
 
 #include "util.h"
 
-#define MAX_SYSCALL		(330)
+struct syscall_errno {
+	int syscall;
+	int error;
```

Comment:
See my comments below about adding an additional field to signify that a given entry should add an argument to the filter.